clang/include/clang/Driver/Options.td

```diff
@@ -3494,6 +3494,12 @@ def fno_struct_path_tbaa : Flag<["-"], "fno-struct-path-tbaa">, Group<f_Group>;
 def fno_strict_enums : Flag<["-"], "fno-strict-enums">, Group<f_Group>;
 def fno_strict_overflow : Flag<["-"], "fno-strict-overflow">, Group<f_Group>,
   Visibility<[ClangOption, FlangOption]>;
+defm init_global_zero : BoolFOption<"init-global-zero",
```

Comment:
`-fzero-init-globals`, maybe? Feel free to ignore if you disagree, though.

Reply:
Thanks @carlocab. As Jean pointed out, there is a gfortran flag `-finit-local-zero`. So the name `-finit-global-zero` was chosen to be similar to it.
Hence, I am not taking this comment. Hope that is OK.